### ISSUE TITLE
Is there a way to specify number of copies?

### ISSUE THREAD

**Issue by mwatz122** [tonyedgecombe/RawPrint#3]
Previously, I had been looping through the PrintRawStream method X number of times to print multiple copies, but that doesn't seem to work on the newer model HP LaserJet M604 series printer that I have recently acquired. Any idea why that would be, or how to correctly print multiple copies? Thanks!

**Comment by tonyedgecombe** [comment]
The copy count is embedded in the print file, rawprint doesn't provide any way to modify that so your only choice is to print repeatedly.

Not sure why your HP isn't working, perhaps you are sending a print file format it doesn't understand.

**Comment by mwatz122** [comment]
Hm, yeah it's strange. Whenever I invoke the method multiple times, I only get one copy (on this specific printer - it works fine on other printers). But I can print multiple copies from other applications such as Adobe Reader without issue. Maybe I will try to modify the code. If I get it working, I would be happy to create a pull request in case including functionality for multiple copies is something you are interested in. Thanks for your input though!

**Comment by tonyedgecombe** [comment]
It might be worth pausing the printer and checking what is spooled, how big the files are and see what happens if you release them singly.

**Comment by mwatz122** [comment]
I tried your suggestion and paused the printing. Both documents are definitely being sent to the printer (very small, only 69.3 KB each in this case). When you release them one at a time, there is a message that pops up on the printer stating "Print data has been received that may not be complete. To print the existing data, press OK." After pressing OK, each document prints. This only works when I release them one at a time, however. Printing normally, I only get one document. These are valid PDF documents, by the way.

After a cursory search online, I was able to find a thread detailing a similar problem here:
https://h30434.www3.hp.com/t5/Printer-Software-and-Drivers/Sending-direct-pdf-print-job-causes-printer-to-error-out/td-p/6005473

Unfortunately, there was no solution listed. I am more so inclined to believe that this is a problem with the printer or printer configuration than with the RawPrint library itself, but thank you for the help nonetheless! I don't suppose you have any other ideas? :/
